intro.tex

```diff
@@ -1,55 +1,67 @@
+
+\subsection{Elements of the Early Science Program}
+
+The \esp consists of the following elements:
+\begin{itemize}
+	\item A series of ``Data Previews'' based on reprocessed data taken during the Rubin Observatory commissioning period. The first, Data Preview 1 (DP1) will be based on ComCam data and the second, Data Preview (DP2) based on LSSTCam data. 
+	\item Template generation prior to the start of regular survey operations based on data collected during the commissioning period with the LSST Camera (LSSTCam) to maximize the number of templates available for Alert Production at the start of the 10-year survey.
+	\item Incremental generation of templates during the first year of survey operations, as and when sufficient images passing quality cuts have been acquired. These templates will then be used immediately during year 1 to generate alerts, thus steadily increasing the number of Alerts generated over the course of year 1. 
+\end{itemize}
+
+Rubin pre-operations is working with the construction project to provide early access to static-sky data products via Data Previews. 
+The first of these, Data Preview 0 (DP0), was released to a group of early adopters from the community in June 2022 and is based on the DESC DC2 simulated dataset, \citep{2021ApJS..253...31L}. 
+Subsequent Data Previews, DP1 and DP2 will be based on the data acquired during  commissioning from ComCam and LSSTCam Science Validation Surveys (SV Surveys) respectively, together with any additional science-quality data taken throughout the full commissioning period.
 
-The \esp is motivated by the fact that in the current baseline there will be no science-ready data products released to the community before \drone.
-This is due to the fact that the templates are needed for Difference Imaging are produced in Data Release Production (DRP) and initially released as part of \drone. 
-Alerts cannot be issued until a template image for a given field and filter is available, hence  in the current baseline, no alerts can be issued until after \drone.
+A key component of the \esp is the generation of templates from both the commissioning data and data collected during the first year of the survey.
+In full survey operations, template images for difference image analysis and alert generation are constructed as part of the annual DRP.
+In order to support alert generation in year 1, Rubin will generate templates from all science-grade data taken during commissioning to provide an initial template library at the start of the 10-year survey, and the incrementally generate templates during year 1 using the best images available and covering as much sky in as many filters as possible.
+Details of the current strategy for alert generation  with incremental templates are given in  \S~\ref{sec:pp}.
```

Comment:
This paragraph seems a bit duplicative of the one in section 2.1 above as well as the bullets in this section.  Suggest looking for opportunities to trim

Reply:
I have removed the text in 2.1 to keep the definition of ES simple